src/mongodb/scores.service.ts

```
@@ -2603,4 +2638,83 @@ export class ScoresService {
+    }
+    return {overallScore,isComprehension};
+  }
+
+  async updateMilestoneLevel(userId: string, subSessionId: string, newMilestoneLevel: string) {
+    return this.scoreModel.updateOne(
+      { user_id: userId, "milestone_progress.sub_session_id": subSessionId },
+      { $set: { "milestone_progress.$.milestone_level": newMilestoneLevel } }
+    );
+  }
+
+  async getComprehensionFromLLM(questionText,studentText,teacherText) {
```

ESLint

[error] Replace `studentText,` with `·studentText,·`

(prettier/prettier)